### DISCUSSION TITLE
FINOS RegTech Council - Door is open - want to get involved?

### DISCUSSION THREAD

**Discussion by PJDiGi** [finos/open-regtech-sig#57] [Ideas]
Greetings! 

The FINOS RegTech Council (RTC) enables Members to accelerate international RegTech adoption through working groups that deliver opensource results. It oversees a portfolio of projects with the support of a programme office.  

We have set two RTC objectives in the initial sprint:  
1.	**Mutualise RegTech deliver**y though derivatives, repo and securities lending regulatory reporting; T+1; and market surveillance 
2.	**Advocate for a global regulatory data dictionary** for US, EU, UK; and support EU ESG data model development 

The programme office, manned by yours truly is kicking off 1:1 discussions now with firms, regulators, associations and those that serve them. We are organising the book of work now and engaged with the Financial Object and RegTech SIGs to drive strategy and adoption. 

By September 2023 we will be holding workshops to discuss the various working group objectives, adoption strategies, milestones and deliverables that will help inform the business case . 

**Interested in joining in or finding out more?** Please contact pj@jwg-it.eu in the first instance. More coming soon...

**Comment by jgavronsky** [comment]
Here is the detailed proposal we've put together for the RegTech Council (RTC). 
Please have a look and let us know what you think, we'd love to hear from you and get you involved!
[FINOS RegTech Council proposal 08-2023.pdf](https://github.com/finos/open-regtech-sig/files/12287626/FINOS.RegTech.Council.proposal.08-2023.pdf)
